### ISSUE TITLE
How to change color for the action text (e.g. Select All & Finish)

### ISSUE THREAD

**Issue by ejaz1977pk** [broakenmedia/MultiContactPicker#31]
By default, these action items turn out as blue (i.e. Azure). Can I change these colors to colorAccent of my project?

**Comment by broakenmedia** [comment]
I don't believe this is currently possible, will look at ways to incorporate it. The text buttons use a custom selector, i probably need to make it replaceable in the builder.
